test.sh

```diff
@@ -1,4 +1,5 @@
 npm install
+node seed.js
 npm run test-mocha
 npm start &
 # node seed.js
```

Comment:
I just read in the example tests, and actually in the `mocha` tests for the models almost every single test that adds data to the database first drops everything, then tests, and finally drops everything again. I guess this is done to avoid problems during the tests.
Nevertheless, seeding before the tests might become useless later.